### DISCUSSION TITLE
Linux Support?

### DISCUSSION THREAD

**Discussion by vagabondHustler** [vagabondHustler/SubSearch#556] [FAQ]
No, sorry https://github.com/vagabondHustler/subsearch/discussions/556#discussioncomment-14717951.

**Comment by itachi-re** [comment]
no support for linux ?

**Reply by vagabondHustler** [reply]
> no support for linux ?

No, I'm not that familiar with Linux. Only got some basic knowledge thru the Windows terminal SSH & WSL.

**Reply by itachi-re** [reply]
alright
